src/PureJuMP/auglag.jl

```diff
@@ -0,0 +1,41 @@
+export auglag
```

Comment:
At the beginning of the JuMP model we also try to add as a commentary as much information as we have on the problem.
See https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/blob/main/src/PureJuMP/arglina.jl for instance. In our case, the name of the problem, reference to the test set, reference to the mentionned paper. Thanks !!!